libnmstate/nm/ipv6.py

```diff
@@ -72,8 +72,6 @@ def get_info(active_connection):
         }
         for address in ipconfig.get_addresses()
     ]
-    if not addresses:
-        return info
```

Comment:
This change will now improperly report disabled IPv6 interface stacks. AFAIK, NM learns once about the current IPv6 enablement and tries to return the interface to that state when the profile is set to `ignore` or removed.

I think there is now a need to fetch this info explicitly from `'/proc/sys/net/ipv6/conf/%s/disable_ipv6' % dev`. We need it anyway for enforcing IPv6 disablement.